### DISCUSSION TITLE
Proposals for small improvements

### DISCUSSION THREAD

**Discussion by eFe-Munoz** [friction2d/friction#100] [Ideas]
Greetings to all. After working these last 2 years with ENVE I shared a series of proposals and Chris Blount has recommended me to put them here, as he comments that this program is a fork of Enve.

I have some small proposals that would facilitate the daily work with ENVE.

OPEN RECENT

    Remove the path of the file to be able to see its name. Eliminate showing the folder path of the file to be able to see its name. This way you can see the file name if it is at the end of a long path (see image).

GENERAL

    Allow to customize keyboard shortcuts.

    Replace the G key with "May" (shift) May to scroll in a straight line (horizontally or vertically). This is a common shortcut in multiple applications.

COLOR

    Allow in preferences to choose the default color mode. If, for example, a person works more comfortably in HSL mode, let that mode remain active instead of having to switch from RGB to HSL every time he/she wants to change a color.

PIVOT

    It would be great if the text pivot alignment options could also be used with the rest of the objects.

TIMELINE

    Object > scale. Possibility to link (icon?) width and height to be able to animate a uniform XY scale with a single value.

    Possibility to select a group of objects in list (Shift or Ctrl) now you can only select several objects one by one.

    Defined space to preview a Timeline zone, so that this zone can be played in block and refine its animation in loop.

    Rename file: Allow double click to rename and not have to delete the current name.

Thanks!

**Comment by toritovitorito** [comment]
I will found really useful animate a uniform XY scale or position. Its really frustraiting to replicate the exact animation keyframes in both parameters haha
